### ISSUE TITLE
Build on version 0.9 doesn’t work

### ISSUE THREAD

**Issue by Kiszaner** [Azoy/Sword#61]
Hi,
I did a checkout on tag 0.9 and tried to build the associated Sword Xcode project, but
the build failed due to missing source files on _Sword/Utils/_ folder: `Imageable.swift` and `Options.swift`. The second one is on git though, but is not included in the xcodeproj.

Output error from Xcode(10.2.1): `error: Build input files cannot be found: '/Users/<user>/Developer/Sword/Sources/Sword/Utils/Imageable.swift', '/Users/<user>/Developer/Sword/Sources/Sword/Types/Options.swift'`

**Comment by Kiszaner** [comment]
Also worth mention that I already tried `swift build` and `swift package update` from the terminal but got an unresolvable dependency graph issue on both situations with the same output:
```Updating https://github.com/Azoy/Sword
Updating https://github.com/daltoniam/Starscream.git
Updating https://github.com/Azoy/Sodium
Updating https://github.com/vapor/sockets.git
error: dependency graph is unresolvable; found these conflicting requirements:

Dependencies: 
    https://github.com/Azoy/Sword @ 0.9.0..<1.0.0

**Comment by zadojart** [comment]
Don't use 0.9, it has been broken for over a year. Use master or, preferably, rewrite.

**Comment by Kiszaner** [comment]
Thanks for the info, but rewrite branch also gives me issues to compile at the moment. Master seems fine, but wasn't sure if it is stable or not, that's why I chose a tagged version.

**Comment by Azoy** [comment]
Hi, sorry about this! I’ve gone ahead and published 0.9.1 to hopefully solve this and future issues. If you still have trouble let me know!
